pkg/instrumentation/helper.go

```diff
@@ -43,3 +44,11 @@ func isAutoInstrumentationInjected(pod corev1.Pod) bool {
 	}
 	return false
 }
+
+func volumeSize(quantity *resource.Quantity) *resource.Quantity {
+	defaultSize := resource.MustParse("150Mi")
```

Comment:
Should this be a constant or in a vars section at the top?

Reply:
done